src/link/Dwarf.zig

```diff
@@ -1035,7 +1035,7 @@ pub fn init(allocator: Allocator, bin_file: *File, format: Format) Dwarf {
         .format = format,
         .ptr_width = ptr_width,
         .dbg_line_header = switch (target.cpu.arch) {
-            .x86_64 => .{
+            .x86_64, .aarch64 => .{
                 .minimum_instruction_length = 1,
```

Comment:
Surely, `.aarch64` should at least have `.minimum_instruction_length = 4`?

Reply:
That's what I was thinking too, but LLVM itself generates the following table:

```
$ zig build-exe -target aarch64-linux-gnu stage2.zig
$ llvm-dwarfdump -a stage2
...
debug_line[0x0002880e]
Line table prologue:
    total_length: 0x00013212
          format: DWARF32
         version: 4
 prologue_length: 0x00000e1b
 min_inst_length: 1
max_ops_per_inst: 1
 default_is_stmt: 1
       line_base: -5
      line_range: 14
     opcode_base: 13
...
```

But setting `minimum_instruction_length` also works and produces correct binaries.